src/utils/staking_utils.ts

```diff
@@ -191,6 +193,18 @@ export const stakingUtils = {
 
         return poolData;
     },
+    getDelegatorEventsFromRaw: (rawDelegatorEvents: RawDelegatorEvent[]): DelegatorEvent[] => {
+        const delegatorEvents: DelegatorEvent[] = rawDelegatorEvents.map(rawEvent => ({
+            eventType: rawEvent.event_type,
+            address: rawEvent.address,
+            blockNumber: Number(rawEvent.block_number),
```

Comment:
If this is undefined it could be a NaN btw, but if it's always 0 or null then we're good.

Reply:
I think it can be relied on to be always null instead of undefined given the structure of the query. However, I changed it to make it be able to take on a null value as opposed to Number(null) = 0